test/core/utils/preload-media.js

```diff
@@ -1,73 +1,84 @@
-describe('axe.utils.preloadMedia', function () {
-  'use strict';
+describe('axe.utils.preloadMedia', () => {
+  const fixtureSetup = axe.testUtils.fixtureSetup;
 
-  var fixture = document.getElementById('fixture');
-  var fixtureSetup = axe.testUtils.fixtureSetup;
-
-  afterEach(function () {
-    fixture.innerHTML = '';
-  });
-
-  it('returns empty array when there are no media nodes to be preloaded', function (done) {
+  it('returns empty array when there are no media nodes to be preloaded', async () => {
     axe._tree = axe.utils.getFlattenedTree(document);
 
-    axe.utils.preloadMedia({ treeRoot: axe._tree[0] }).then(function (result) {
-      assert.equal(result.length, 0);
-      done();
-    });
+    const result = await axe.utils.preloadMedia({ treeRoot: axe._tree[0] });
+    assert.equal(result.length, 0);
   });
 
-  it('returns empty array when <audio> has no source', function (done) {
+  it('returns empty array when <audio> has no source', async () => {
     fixtureSetup('<audio autoplay="true" controls></audio>');
 
-    axe.utils.preloadMedia({ treeRoot: axe._tree[0] }).then(function (result) {
-      assert.equal(result.length, 0);
-      done();
-    });
+    const result = await axe.utils.preloadMedia({ treeRoot: axe._tree[0] });
+    assert.equal(result.length, 0);
   });
 
-  it('returns empty array when <video> has no source', function (done) {
+  it('returns empty array when <video> has no source', async () => {
```

Comment:
Should be updated to set autoplay, otherwise the test would pass regardless of the source being missing